Week1/homework/app.js

```diff
@@ -3,9 +3,173 @@
 {
   const bookTitles = [
```

Comment:
These `bookTitles` are unused in your code (other than in a console.log). See my message in slack how these are supposed to be used:

https://hackyourfuture.slack.com/archives/CDU4A4W9L/p1547224613021500